util/tls_go17.go

```diff
@@ -12,7 +12,6 @@
 // limitations under the License.
 
 //go:build go1.7 && !go1.8
-// +build go1.7,!go1.8
```

Comment:
And here I think we can safely remove the whole file, it was used only when building with go 1.7.

Reply:
Thanks, done!